nilearn/datasets/description/destrieux_surface.rst

```diff
@@ -0,0 +1,20 @@
+Destrieux Surface
+
+
+Notes
+-----
+Anatomical parcellation of the cortex based on sulcal depth. This atlas is
+based on the fsaverage5 surface data.
```

Comment:
Maybe: "Anatomical parcellation of the cortex based on sulco-gyral patterns (Destrieux et al, 2010) distributed with Freesurfer (Dale et al, 1999, Fischl et al, 1999) in fsaverage5 standard surface space."